lib/LANraragi/Utils/Tags.pm

```diff
@@ -0,0 +1,122 @@
+package LANraragi::Utils::Tags;
```

Comment:
This new module should be added to the list in `tests/modules.t`.   
(It's already quite well tested by the unit tests you've written, but all modules are in this list for basic perl interpreter checks on CI.)

Reply:
No problem!